.github/pull_request_template.md

```diff
@@ -1,3 +1,9 @@
 ## Summary & Motivation
 
 ## How I Tested These Changes
+
+## Did you update the changeset?
```

Comment:
Nit: should be "Did you add a changeset?"

Reply:
[a45f738](https://github.com/tkhq/sdk/pull/164/commits/a45f73848d7d2caf78d0c203f0333f64d4b407a6)